ember/app/controllers/map/moderations/index.js

```diff
@@ -40,13 +56,36 @@ export default class extends ModerationController {
   deny(moderation) {
     const id = moderation.get('id');
     const elem = document.querySelector(`li[data-mod-id="${id}"]`);
-    const development = moderation.get('development.name');
     const user = moderation.get('user.fullName');
+    const development = moderation.get('development.name');
 
     this.get('notifications').error(`You have denied an edit from ${user} for ${development}`);
-    
+
     moderation.destroyRecord();
     elem.parentNode.removeChild(elem);
   }
 
+
+  @action
+  unflag(development) {
+    const id = development.get('id');
+    this.set('unflagging', id);
+
+    development.set('flag', false);
+    development.save()
+    .then(() => {
+      this.get('notifications').show('This development has been unflagged.');
+
+      const elem = document.querySelector(`li[data-flag-id="${id}"]`);
```

Comment:
We're using Ember. Why are we manually removing a node with JavaScript?

Reply:
Because these developments aren't being pulled in the standard Ember way. We are using my filtering endpoint instead of expanding `Developments#index` in Rails to allow this type of filtering.

Reply:
I might be able to shove this off to the route which would fix the issue.

Reply:
But you already have the data inside of a controller? You should still be able to set variables that would affect the handlebars template rather than directly manipulating the DOM.